c7n/resources/ecs.py

```diff
@@ -294,6 +294,86 @@ class ServiceTaskDefinitionFilter(RelatedTaskDefinitionFilter):
              - type: stop
     """
 
+@ECSCluster.filter_registry.register('storage')
```

Comment:
Wondering about the naming here. We have the existing `ebs` filter for `aws.ec2` which serves this case at the instance level. So `ebs` or `volume` might be more in line with existing filters, but `storage` being more neutral might be a positive...

Reply:
This was originally named I had a vision of covering the various ECS task storage options beyond EBS (eg EFS & FSX) That obviously didn't get done in this PR and perhaps a more specific name would be appropriate.